lib/ServerDate.js

```diff
@@ -122,7 +115,7 @@ Object.defineProperty(ServerDate, "synchronizationIntervalDelay", {
 
 // After the initial synchronization the two clocks may drift so we
 // automatically synchronize again every synchronizationIntervalDelay.
-ServerDate.synchronizationIntervalDelay = 60 * 60 * 1000; // ms
+ServerDate.synchronizationIntervalDelay = 10 * 60 * 1000; // ms, 10 minutes
```

Comment:
Not that much, in most cases, in fact. But considering how easy is to sync (anyway, we do it 10 times after initial load) and the fact there are environments where the drift could be significant (imagine running browser within virtual machine having poor syncronisation with the host) I think we can afford doing sync more often. What do you think?